# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/mozilla/meta/editor/delete-space-after-double-click-selection.html.ini
# Commit: bdbed5920aeb
# Full Hash: bdbed5920aeb2adc75f03ac0f6c427b3ac456395
# Author: Jan-Niklas Jaeschke <jjaschke@mozilla.com>
# Date: 2022-10-26 03:47:46
# Regressor Bug: 1783641
# File Overlap Count: 2
# Description:
#   Bug 1783641: Deleting a selected word deletes surrounding whitespace on MacOS. r=masayuki
#   
#   Native behaviour on MacOS dictates one whitespace being removed after double-clicking a word and pressing delete.
#   This behaviour is achieved by saving the information that the selection is created by doubleclick to the `nsFrameSelection`
#   and using it in the `DeleteRangeTransaction`, where the range is extended by one whitespace character before or after the range.
# ==============================================================================

diff -r a32dbcfbd95f -r bdbed5920aeb testing/web-platform/mozilla/meta/editor/delete-space-after-double-click-selection.html.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/mozilla/meta/editor/delete-space-after-double-click-selection.html.ini	Tue Oct 25 07:00:26 2022 +0000
@@ -0,0 +1,6 @@
+[delete-space-after-double-click-selection.html]
+  [DIV: Only one whitespace character should be removed when there are multiple.]
+    expected: FAIL
+
+  [DIV: Only one whitespace character should be removed when there are multiple whitespaces and the deleted range is the end of the string.]
+    expected: FAIL